### PR TITLE
Generate a self-signed certificate for SSL

### PR DESCRIPTION
**Pull request by prettyClouds**
Tracker #167072556

Co-authored-by: Murtuza Boxwala <mboxwala@pivotal.io>
Co-authored-by: Kamilla Aslami <kaslami@pivotal.io>

### CODE REVIEW

**Comment by prettyClouds** [comment]
@pivotal-jbarrett @balesh2 Could you please review this PR?

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ out
 .gradle
 build/
 output*/
+selfsigned.jks
```
Comment (pivotal-jbarrett) [inline .gitignore]: If the benchmarks are generating this in the runtime directory we shouldn't need to exclude this anymore.

File: harness/build.gradle [hunk 2]
```diff
@@ -55,6 +55,12 @@ dependencies {
     testCompile(group: 'org.assertj', name: 'assertj-core', version: project.'assertj-core.version')
 }
 
+compileJava {
+    options.fork = true
+    options.forkOptions.executable = 'javac'
+    options.compilerArgs << '-XDignore.symbol.file'
```
Comment (pivotal-jbarrett) [inline harness/build.gradle]: This is sort of a deprecated way to ignore warnings. I would just pull it. We have lots of similar warnings in our main sources.

File: harness/src/main/java/org/apache/geode/perftest/jvms/JVMLauncher.java [hunk 3]
```diff
@@ -85,6 +90,13 @@ public void run() {
     command.add("-D" + RemoteJVMFactory.RMI_PORT_PROPERTY + "=" + rmiPort);
     command.add("-D" + RemoteJVMFactory.JVM_ID + "=" + jvmConfig.getId());
     command.add("-D" + RemoteJVMFactory.OUTPUT_DIR + "=" + jvmConfig.getOutputDir());
+    String withSslArg = System.getProperty("withSsl");
+    if (withSslArg != null) {
```
Comment (pivotal-jbarrett) [inline harness/src/main/java/org/apache/geode/perftest/jvms/JVMLauncher.java]: Use
```java
if (Boolean.getBoolean("withSsl")) {
...
}
```

File: infrastructure/scripts/aws/README.md [hunk 4]
```diff
@@ -179,17 +179,6 @@ monitor the test.
...
-## Prerequisites
-* You must have fulfilled the prerequisites at the beginning of this doc
-* Generate a self-signed SSL certificate using the keytool command:
-  * `keytool -genkey -keyalg RSA -alias tomcat -keystore selfsigned.jks -validity 365 -keysize 2048`
-  * The keystore password must be `123456`
-* Copy the generated certificate to the AWS VMs using the following command:
-  * `./copy_to_cluster.sh -tag <clusterTag> -- <path to selfsigned.jks> /home/geode/selfsigned.jks`
-  * The destination path must be `/home/geode/selfsigned.jks`
-
-## Running in AWS
```
Comment (pivotal-jbarrett) [inline infrastructure/scripts/aws/README.md]: Looks like you accidentally wiped out the Running in AWS header.

File: harness/build.gradle (hunk 2 above)
Comment (prettyClouds) [reply harness/build.gradle]: i double checked, and without these the build generates an error.  
```error: package sun.security.tools.keytool does not exist 
import sun.security.tools.keytool.CertAndKeyGen;```

without this flag, javac won't include some of the older sun classes

File: infrastructure/scripts/aws/README.md (hunk 4 above)
Comment (prettyClouds) [reply infrastructure/scripts/aws/README.md]: That was on purpose...it seems redundant in the context of the file.

File: .gitignore (hunk 1 above)
Comment (prettyClouds) [reply .gitignore]: we renamed the file to make it more obvious that it's temporary, but it is still generated in the directory the jar/gradle is run from.

File: harness/src/main/java/org/apache/geode/perftest/jvms/JVMLauncher.java (hunk 3 above)
Comment (prettyClouds) [reply harness/src/main/java/org/apache/geode/perftest/jvms/JVMLauncher.java]: done

**Pull request merged**